composer.json

```diff
@@ -32,7 +32,7 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.19 || ^3.8",
         "mf2/mf2": "^0.5.0",
-        "phpstan/phpstan": "^1.11",
+        "phpstan/phpstan": "~1.12.2",
```

Comment:
Unfortunately, we cannot use single `composer.lock` since we support wide range of PHP versions that require different `php-cs-fixer` versions.

Reply:
Does it makes sense at all to run `php-cs-fixer` with multiple PHP versions? There is a single code base / syntax.

Reply:
FWIW, we have been using PHPCS + PHPCBF on https://github.com/FreshRSS/FreshRSS/ with multiple PHP versions for years without such problems

Reply:
The issue is that Composer will always try to install all development dependencies and installation will fail for incompatible PHP versions.